controller/DonationController.js

```diff
@@ -41,6 +42,44 @@ exports.makeDonation = catchAsyncError(async (req, res) => {
   }
 });
 
+/**
+ * @description Donate larger sums of money (NGN 500,001 and upwards)
+ * @route POST /v1/donation/bulk-sum
+ * @access PUBLIC
+ */
+exports.createManualDonation = catchAsyncError(async (req, res) => {
+  /**
+   * @todo Handle validation for req.body
+   */
+  const { name, email, phone_number, ...referralDetails } = req.body;
+  const newReferral = await Referral.createReferral({
+    ...referralDetails,
+    transactionCompleted: true
+  });
+
+  if (!newReferral) {
+    return res.status(400).json({
+      status: false,
+      message: "Could not record donation, possibly duplicate tx_ref"
+    });
+  }
+
+  const newDonation = await Donation.createManualDonation(req.body);
+
+  if (!newDonation) {
+    return res.status(500).json({
+      status: false,
+      message: "Could not create manual donation entry, please try again."
+    });
+  }
+
+  Logger.info("Manual donation created")
```

Comment:
@Chiemezuo, I think it will be best to add some details about the donation as well like the transaction reference. So something like:

`Logger.info(`Manual donation created with transaction id: `${referralDetails.trx_ref})